recipes-nemomobile/tut-dev/tut-dev_git.bb

```diff
@@ -11,8 +11,10 @@ S = "${WORKDIR}/git"
 
 inherit waf
 
-B = "${S}"
+B = "${WORKDIR}/build"
```

Comment:
Isn't that the default value of B?

Reply:
https://github.com/openembedded/openembedded-core/blob/master/meta/classes/waf.bbclass#L8

Reply:
This is indeed correct. I have verified this locally. I shall remove this in a subsequent rebase.